gsoc2018-bharat/src/create_db.py

```diff
@@ -0,0 +1,69 @@
+import json
+from gensim.models import FastText
+from gensim.models import KeyedVectors
+import torch
+from torch.autograd import Variable
+import numpy as np
+import Encoder
+import logging
+import sys
+
+
+logging.basicConfig(format='%(levelname)s : %(message)s', level=logging.INFO)
+logging.root.level = logging.INFO
+model = torch.load('model/description_encoder')
+# wv = KeyedVectors.load_word2vec_format('model/glove.6B.100d.txt.word2vec',
+#                                        binary=False)
+m = FastText.load('model/entity_fasttext_n100')
+wv = m.wv
+del m
+punctuation = '!"#$%&\'()*+,.:;<=>?@[\\]^`{|}~'
+table = str.maketrans('', '', punctuation)
+
+
+def main(model, wv, abstracts_file, db_file):
```

Comment:
It feels like a bunch of this code lives somewhere else. Have I seen this before. Also you need to decide whether you are using print statements to debug or not